pkg/model/components/etcdmanager/model.go

```diff
@@ -207,7 +209,12 @@ func (b *EtcdManagerBuilder) buildPod(etcdCluster kops.EtcdClusterSpec) (*v1.Pod
 
 	// TODO: pull from bundle
 	bundle := "(embedded etcd manifest)"
-	manifest = []byte(defaultManifest)
+
+	img := os.Getenv("ETCD_MANAGER_IMAGE")
+	if img == "" {
+		img = "kopeio/etcd-manager:3.0.20201202"
```

Comment:
the etcd-manager image can be overridden by [API field](https://pkg.go.dev/k8s.io/kops/pkg/apis/kops#EtcdClusterSpec) `.spec.etcdClusters[*].image` so we probably dont need this override support here.

btw https://github.com/kopeio/etcd-manager/pull/372 was merged so the next etcd-manager release will include azure support 👍🏻

Reply:
The next etcd-manager release was already merged. If there are any other Azure related changes there, please create a PR. In any case, this is something that should not be merged in main branch.